apps/console/src/features/users/components/wizard/bulk-import-user-wizard.tsx

```diff
@@ -1535,6 +1536,29 @@ export const BulkImportUserWizard: FunctionComponent<BulkImportUserInterface> =
                         !showManualInviteTable
                             ? (
                                 <>
+                                    {
+                                        !userConfig?.enableBulkImportSecondaryUserStore
+                                        && (
+                                            <Grid.Row columns={ 1 } className="mb-0 pb-0">
+                                                <Grid.Column mobile={ 16 } tablet={ 16 } computer={ 16 }>
+                                                    <Alert severity="info">
+                                                        <Trans
+                                                            i18nKey={
+                                                                "console:manage.features.user.modals." +
+                                                                "bulkImportUserWizard.wizardSummary.userstoreMessage"
+                                                            }
+                                                            tOptions={ {
+                                                                userstore: camelCase(userstore)
+                                                            } }
+                                                        >
+                                                    The created users will be added to
+                                                    the <b>{ camelCase(userstore) }</b> user store.
```

Comment:
Shall we left-indent this text?

Reply:
Improved with 20838279b40f42b0e2309227901e42733e073355